pkg/api/const.go

```diff
@@ -30,6 +30,7 @@ const (
 	CoreOS          Distro = "coreos"
 	AKS             Distro = "aks"
 	AKSDockerEngine Distro = "aks-docker-engine"
+	AKS1604         Distro = "aks-16.04"
```

Comment:
I would advocate we move forward instead of retroactively changing distro values. (I infer that the "aks" distro will become 18.04-enabled after this change). How about this:

- add two new distros:
  - `aks-ubuntu-16.04`
  - `aks-ubuntu-18.04`
- the `aks` distro becomes an "alias" for `aks-ubuntu-16.04`

Going forward, we will commit to using more descriptive/specific distro specs for VHD images. But let's not retroactively change the functional thing that the existing `aks` distro provides (i.e., Ubuntu 16.04 LTS).

Sorry if I'm not interpreting intent correctly!

Reply:
That makes sense. My intention here was that "aks" was default and 16.04 was the feature flagged one, just like we did with moby and docker-engine.